CompilerSource/backend/GameData.cpp

```diff
@@ -459,7 +459,7 @@ ImageData::ImageData(const Image &img):
 ImageData::ImageData(int w, int h, const uint8_t *data, size_t size):
     width(w), height(h), pixels(data, data + size) {}
 
-GameData::GameData(deprecated::JavaStruct::EnigmaStruct *es): filename(es->filename ?: "") {
+GameData::GameData(deprecated::JavaStruct::EnigmaStruct *es): filename(es->filename ? es->filename : "") {
```

Comment:
fundies, why did you do that, the Elvis operator is alive and well?

Reply:
You can make this work under MSVC if you want.
https://docs.microsoft.com/en-us/cpp/build/reference/zc-ternary?view=vs-2019